connext_cmake_module/connext_cmake_module-extras.cmake

```diff
@@ -14,4 +14,5 @@
 
 # copied from connext_cmake_module/connext_cmake_module-extras.cmake
 
+find_package(ament_cmake REQUIRED)
```

Comment:
Hmm, that function should probably be name spaced. It would suck if someone else wrote a macro called that which worked differently from the one in ament and then tried to use it in conjunction with other ament stuff.

Obviously not something that needs to be addressed in this pr.